core/block.go

```diff
@@ -26,6 +26,8 @@ type Header struct {
 	Timestamp *felt.Felt
 	// The version of the Starknet protocol used when creating this block
 	ProtocolVersion string
+	// The status of the block
+	Status string
```

Comment:
core.Block should not have this.

we can decide on a blocks status just by looking at the `L1ChainHeight` we have in database.
```
if L1ChainHeight > block.Number {
 return "L1_VERIFIED"
} else {
 return "L2_VERIFIED"
}

```

Reply:
Yeah, it is definitely OK. You can use a GatewayClient, this is just a placeholder anyways.